Dockerfile

```diff
@@ -44,9 +50,29 @@ ARG telemetryServer=telemetry-server
 #
 FROM ${POSTGRES_BASE}:${POSTGRES_MAJOR} AS telemetry-postgres
 
-# add the telemetry init scripting
+#
+# Build the customised telemetry-tools image
+#
+FROM ${TOOLS_BASE}:${TOOLS_MAJOR} AS telemetry-tools
```

Comment:
We'll need to rename the SUSE/telemetry's telemetry-tools image to telemetry-client-tools ;-)